vllm/engine/arg_utils.py

```diff
@@ -98,7 +98,9 @@ class EngineArgs:
     speculative_disable_by_batch_size: Optional[int] = None
     ngram_prompt_lookup_max: Optional[int] = None
     ngram_prompt_lookup_min: Optional[int] = None
-
+    speculative_draft_token_sampling_method: str = 'rejection_sampler'
+    typical_acceptance_sampler_posterior_threshold: float = 0.09
+    typical_acceptance_sampler_posterior_alpha: float = 0.3
```

Comment:
seems we default to None for other speculative hyperparams that aren't enabled by default. wdyt about keeping it consistent?

Reply:
Defaulting to None now.